packages/atproto_client/models/string_formats.py

```diff
@@ -1,3 +1,14 @@
+"""AT Proto string format validation.
+
+*Note*: These formats are a working empirical understanding of the following resources:
+
+- https://atproto.com/specs/lexicon
+
+- https://github.com/bluesky-social/atproto/tree/main/interop-test-files/syntax
+
+see https://github.com/MarshalX/atproto/pull/406 for more details.
```

Comment:
could we create string_formats.md (same as we have, for examle, for `auth.md` and write all the details? instead of mentioning Pull Request. 

also it would be great to provide here a little code snippet how to use this optonal validation; as we have in the main readme, for example